### PR TITLE
📚 Docs Refresh: MySQL Source

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Reorganization
- Removing superfluous information

### CODE REVIEW

File: docs/integrations/sources/mysql.md [hunk 1]
```diff
@@ -64,16 +11,24 @@ If you do not see a type in this list, assume that it is coerced into a string.
 | SSL Support | Yes |  |
 | SSH Tunnel Connection | Coming soon |  |
 | Namespaces | Yes | Enabled by default |
+| Arrays | Yes | Byte arrays are not supported |
```
Comment (sherifnada) [inline docs/integrations/sources/mysql.md:14]:
```suggestion
| Arrays | Yes | Byte arrays are not supported yet | 
```

File: docs/integrations/sources/mysql.md [hunk 2]
```diff
@@ -64,16 +11,24 @@ If you do not see a type in this list, assume that it is coerced into a string.
...
+| Arrays | Yes | Byte arrays are not supported | 
+
+The MySQL source does not alter the schema present in your database. Depending on the destination connected to this source, however, the schema may be altered. See the destination's documentation for more details.
+
+## Troubleshooting
+
+There may be problems with mapping values in MySQL's datetime field to other relational data stores. MySQL permits zero values for date/time instead of NULL which may not be accepted by other data stores. To work around this problem, you can pass the following key value pair in the JDBC connector of the source setting `zerodatetimebehavior=Converttonull`.
+
+## Getting Started (Airbyte Cloud)
+On Airbyte Cloud, only TLS connections to your MySQL instance are supported in source configuration. Other than that, you can proceed with the open-source instructions below.
```
Comment (sherifnada) [inline docs/integrations/sources/mysql.md:23]:
```suggestion
On Airbyte Cloud, only TLS connections to your MySQL instance are supported. Other than that, you can proceed with the open-source instructions below.  
```

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/mysql.md [hunk 3]
```diff
@@ -64,16 +11,24 @@ If you do not see a type in this list, assume that it is coerced into a string.
 | SSL Support | Yes |  |
 | SSH Tunnel Connection | Coming soon |  |
```
Comment (marcosmarxm) [inline docs/integrations/sources/mysql.md:12]: Supported now

**Review by marcosmarxm — APPROVED** [review]
LGTM

File: docs/integrations/sources/mysql.md (hunk 3 above)
Comment (avaidyanatha) [reply docs/integrations/sources/mysql.md]: Done!

**Pull request merged**